airflow/api_fastapi/common/parameters.py

```diff
@@ -239,7 +239,7 @@ def to_orm(self, select: Select) -> Select:
             raise ValueError(f"Cannot set 'skip_none' to False on a {type(self)}")
 
         if self.value is None:
-            return select
+            self.value = self.get_primary_key_string()
```

Comment:
I wonder if it’s easier for people to reason with if this defaults to the first item in `allowed_attrs` instead.

Reply:
True. This is not the only place where we do it, I will merge like this to at least unblock main.

But I definitely keep this suggestion in mind. (Even if for now the 'PK' is retrieved, and displayed in the documentation as a normal filter, it's just on the developer's side that it might be more complicated to understand)
![Screenshot 2024-11-26 at 17 47 09](https://github.com/user-attachments/assets/048051b2-fba7-4b42-ac0c-68b9a30c7d89)

We still have some improvement to do in that area, WIP. (we are missing the exhaustive list of possible values in the documentation at the moment too)